ui/common/css/theme/_default.scss

```diff
@@ -1,156 +1,118 @@
-@import '../abstract/functions';
+@import 'gen/mix';
 
-/* Colors */
+// this may not import anything from outside common/css/colors
 
-@function c-dimmer($color, $weight: 17%) {
+@function clear($color, $weight: 20%) {
   @return mix(#fff, $color, $weight);
 }
 
-@function c-clearer($color, $weight: 17%) {
+@function dim($color, $weight: 20%) {
   @return mix(#000, $color, $weight);
 }
 
-@function set-alpha($color, $alpha) {
-  // set alpha channel if it already exists, add it if not
-  @return rgba(red($color), green($color), blue($color), $alpha);
+$site-hue: 37 !default;
+
+/* top level $c-variables must be valid css format (rgba, hsla, hex) and are mixable by ui/build
```

Comment:
This should look vaguely familiar too. once more, if there are questions check the readme.md.